erizo/src/erizo/DtlsTransport.cpp

```diff
@@ -10,7 +10,8 @@
 
 #include "./SrtpChannel.h"
 #include "rtp/RtpHeaders.h"
-#include "lib/LibNiceInterface.h"
+#include "./NiceConnection.h"
+#include "./NicerConnection.h"
```

Comment:
yep 😄 , I renamed NiceConnection to LibNiceConnection anyway to avoid confusions in the future, as Pedro suggested before.